docs/src/gui/qtvcp-vismach.adoc

```diff
@@ -133,12 +133,12 @@ c.newpin("joint1", hal.HAL_FLOAT, hal.HAL_IN)
 c.ready()
 ----
 
-You can choose this differents in the function that take these entries:
+You can choose this difference in the function that takes these entries:
```

Comment:
Wasn't certain about this one

Reply:
I think maybe "You can choose this difference..." becomes "You can select between the two options in the functions that take these entries:"

Reply:
Done.